addons/knowledge/static/src/js/knowledge_renderers.js

```diff
@@ -135,6 +159,25 @@ const KnowledgeArticleFormRenderer = FormRenderer.extend(KnowledgeTreePanelMixin
         });
     },
 
+    /**
+     * Update the chatter.
+     * Called by the @see KnowledgeController when a reload is initiated by the
+     * chatter.
+     */
+    async updateChatter() {
```

Comment:
update or reload ?

Reply:
From what I understand (but I may be wrong):
In owl terminology there is no reload, this functions guarantees that the chatter has the lastest information and is fully operational (updated, rendered and mounted), and in this case, this function will be called all (most?) of the time when the chatter is already rendered and mounted, but does not have updated props. The mount operation may even be useless, I'll check if I can remove it.

The reload comes from an event dispatched in `chatter.js`, and in my opinion, unless some information has to be displayed in the view related to the message thread (other than the chatter itself), a full reload of the view is useless/detrimental (it's my guess that "keepChanges=true" is an indicator of this)

(But I'm still new to this so maybe I should call this function differently)